modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/SequencesOnDisplay.scala

```diff
@@ -128,11 +128,13 @@ final case class SequencesOnDisplay(tabs: Zipper[SeqexecTab]) {
         val curTableState = tab
           .map(_.tableState)
           .getOrElse(StepsTable.State.InitialTableState)
+        val s = tag[InstrumentSequenceTab.LoadedSV][SequenceView](x).asRight
+        val seq = tab.map{t => if (t.isComplete) tag[InstrumentSequenceTab.CompletedSV][SequenceView](x).asLeft else s}
         val stepConfig   = tab.flatMap(_.stepConfig)
         val selectedStep = tab.flatMap(_.selectedStep)
         InstrumentSequenceTab(
           x.metadata.instrument,
-          tag[InstrumentSequenceTab.LoadedSV][SequenceView](x).asRight,
+          seq.getOrElse(s),
```

Comment:
The logic here with `s` and `seq` is kind of weird … could you simplify? Something like this maybe?

```scala
lazy val left  = tag[InstrumentSequenceTab.CompletedSV][SequenceView](x)
lazy val right = tag[InstrumentSequenceTab.LoadedSV][SequenceView](x)
val seq = tab.filter(_.isComplete).as(left).toLeft(right)
```

Reply:
Sure, sounds like a good suggestion